docs/md/usage-retry.md

```diff
@@ -1,3 +1,52 @@
+# Retry Workflows in Terra
+
+WFL has `/retry` endpoint
+that selects workflows
+by their completion status
+and re-submits them.
+
+The following `curl` shell command
+finds the workflows
+with the Cromwell status `"Failed"`
+in the workload with `$UUID`
+and resubmits them for processing.
+
+```bash
+WFL=https://gotc-prod-wfl.gotc-prod.broadinstitute.org/api/v1/workload
+AUTH="Authorization: Bearer $(gcloud auth print-access-token)"
+UUID=0d307eb3-2b8e-419c-b687-8c08c84e2a0c # workload UUID
+
+curl -X POST -H "$AUTH" $WFL/$UUID/retry --data '{"status":"Failed"}' | jq
+```
+
+The typical
+[Cromwell statuses](https://github.com/broadinstitute/wfl/blob/c30e77450926d25b085696b08a306775cd244ea1/api/src/wfl/service/cromwell.clj#L14)
+to drive the `/retry` API are:
+- `"Aborted"`
+- `"Failed"`
+
+`"Aborted"` and `"Failed"` are terminal,
+and likely failure,
+statuses for workflows.
+
+Specifying other Cromwell statuses
```

Comment:
Howdy, my [implementation PR](https://github.com/broadinstitute/wfl/pull/482) is ready for review.  What I want to note is that when we retry a workflow, we have to resubmit all workflows from the same submission since we can't submit subsets of snapshots yet. So if you're retrying a failed workflow in a still-running submission, note that all workflows in the submission including those still running will be retried.